stable/sonarqube/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 0.9.1
+version: 0.9.2
```

Comment:
Please update the minor version as this adds a feature.

Reply:
Hey,

Thats done now. 

Thanks